variables.tf

```diff
@@ -216,6 +223,24 @@ variable "kubernetes_version" {
   default     = null
 }
 
+variable "load_balancer_profile" {
+  type        = bool
+  description = "(Optional) A load_balancer_profile block. This can only be specified when load_balancer_sku is set to standard."
+  default     = null
+  nullable    = true
+}
+
+variable "load_balancer_sku" {
+  type        = string
+  description = "(Optional) Specifies the SKU of the Load Balancer used for this Kubernetes Cluster. Possible values are basic and standard. Defaults to standard."
```

Comment:
Could we backquote the enum value as the [document](https://github.com/hashicorp/terraform-provider-azurerm/blob/f8e47e77eaa8baf00918d6172edefe97fdfaaa7b/website/docs/r/kubernetes_cluster.html.markdown?plain=1#L555) did? It could be:

```hcl
description = "(Optional) Specifies the SKU of the Load Balancer used for this Kubernetes Cluster. Possible values are `basic` and `standard`. Defaults to `standard`."
```